azerothcore/azerothcore-wotlk
# Fix(Core/Script): Molten Core: Magmadar

Co-Author: @sanctum32 (Code)
Co-Author: @t1ti (Research)
Co-Author: @Hacki95 (Coordination)



## Changes Proposed:
- Modify the timers of all spells to the researched values.
- Add missing aura Magma-Spit (19449)
- Add missing spell Lava Bomg (19411)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5457

## SOURCE:

- Research done in #5457

## Tests Performed:

- Applied fix
- Fought Magmadar


## How to Test the Changes:


1. Apply the patch
2. .tele moltencore
3. Fight Magmadar
4. Check the timers and spells

## Known Issues and TODO List:


- Core Hounds are not blizzlike yet. They should use a spell for the feign death


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
